GitHubData/Factories/HTTPClientFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿// <copyright file="HTTPClientFactory.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace GitHubData.Factories;
+
+/// <summary>
+/// Creates an <see cref="IHttpClient"/> for making HTTP request.
```

Comment:
Refactor code docs to below:

```suggestion
/// Creates an <see cref="IHttpClient"/> for making HTTP requests.
```